go-controller/pkg/ovn/ovn_test.go

```diff
@@ -184,9 +184,9 @@ func (o *FakeOVN) init(nadList []nettypes.NetworkAttachmentDefinition) {
 		o.fakeRecorder, o.wg)
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 	o.controller.multicastSupport = config.EnableMulticast
-	o.controller.clusterLoadBalancerGroupUUID = types.ClusterLBGroupName + "-UUID"
-	o.controller.switchLoadBalancerGroupUUID = types.ClusterSwitchLBGroupName + "-UUID"
-	o.controller.routerLoadBalancerGroupUUID = types.ClusterRouterLBGroupName + "-UUID"
+
+	setupCOPP := false
```

Comment:
hmm why are we running tests with setupCopp=false?
we don't provide that option in reality, ovnk is always setup with copp=true
I'd say tests should also just run with true?